istresearch/galton
# Add statsd support

This PR adds statsd support for galton.

## Testing

1) Run your dd-agent locally

For OS X:

    $ sudo datadog-agent start

For Ubuntu:

    $ sudo service datadog-agent start

2) Run the tests

3) Observe the galton metrics in DataDog
